smac/tae/execute_ta_run_hydra.py

```diff
@@ -3,25 +3,26 @@
 from smac.configspace import Configuration
 from smac.tae.execute_ta_run_old import ExecuteTARunOld
 from smac.tae.execute_ta_run_aclib import ExecuteTARunAClib
-from smac.tae.execute_ta_run_aclib import ExecuteTARun
+from smac.tae.base import BaseRunner
 from smac.tae.execute_func import ExecuteTAFuncDict
 from smac.tae.execute_func import ExecuteTAFuncArray
-from smac.tae.execute_ta_run_old import StatusType
+from smac.tae import StatusType
+from smac.tae.serial_runner import SerialRunner
 
 __copyright__ = "Copyright 2018, ML4AAD"
 __license__ = "3-clause BSD"
 __maintainer__ = "Marius Lindauer"
 
 
-class ExecuteTARunHydra(ExecuteTARun):
+class ExecuteTARunHydra(SerialRunner):
 
     """Returns min(cost, cost_portfolio)
     """
 
     def __init__(
         self,
         cost_oracle: typing.Mapping[str, float],
-        tae: typing.Type[ExecuteTARun] = ExecuteTARunOld,
+        tae: typing.Type[BaseRunner] = ExecuteTARunOld,
```

Comment:
Oh yeah, as the code is right now yes, I wanted to set the base class in case someone implements a runner inheriting from BaseRunner which is more general than SerialRunner. But then I will move to SerialRunner, and this can be changed if needed